docs/sphinx/dev_guide/ci.rst

```diff
@@ -8,13 +8,76 @@
 
 .. _ci:
 
-**********************
-Continuous Integration
-**********************
+************************************
+Continuous Integration (CI) Testing
+************************************
+
+The RAJA project employes multiple tools to run its tests for each GitHub
+*pull request*, all of which must pass before the pull request can be merged.
+These tools include:
+
+  * **Travis CI.** This runs builds for a Linux system for multiple versions of
+    the GNU and clang compilers, and one version of the Intel, nvcc (CUDA), and 
+    HIP compilers. RAJA tests are run each non-GPU build.
+  * **Appveyor.** This runs builds and tests for a Windows environment for two
+    versions of the Visual Studio compiler.
+  * **Gitlab CI.** Runs builds and tests on platforms in the Livermore
```

Comment:
I am a strong proponent of the oxford comma :)
"Runs, builds, and tests on..."
(typo)

Reply:
I agree wih the oxford comma point. but this is not a list. I changed the wording to clarify the point.

Reply:
Gasp! Yay! Oxford commas are the best!
Sounds good. Thanks!